marketdata/stream/entities.go

```diff
@@ -200,3 +195,14 @@ func (e errorMessage) Error() string {
 
 	return e.msg
 }
+
+type CryptoFuturesPricing struct {
```

Comment:
This is inconsistent with the REST endpoint now: that's called `LatestCryptoPerpPricing`. This should be called `CryptoPerpPricing` then, too.